src/hotspot/share/gc/shenandoah/shenandoahFreeSet.inline.hpp

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
I made some of these services "public" so that googletest can exercise the APIs from test/hotspot/gtest/gc/shenandoah/shenandoahSimpleBitMap.cpp.  Please advise if there's a better way.